exercises/palindrome-products/canonical-data.json

```diff
@@ -144,7 +140,7 @@
               "max": 1
             },
             "expected": {
-                "error": "invalid input: min is 10000 and max is 1"
```

Comment:
I agree that the old `error` was not useful, and that `min must be <= max` is a significant improvement

This change can be in a separate commit from the change about the cases resulting in no factors, since these two changes are dealing with completely different things. I might go so far as to say they *should* be in separate commits.

Reply:
You are right in both cases, so I will close this PR for now and open two separate ones instead.